2024-02-09 09:14:51,998 - process_api - INFO - run schema: "test_regular"
2024-02-09 09:14:51,999 - process_api - INFO - run process: "main"
2024-02-09 09:14:51,999 - process_api - INFO - run step: "start"
2024-02-09 09:14:51,999 - process_api - INFO - perform navigate: ${state.server}/#dashboard/RegularAsset
2024-02-09 09:14:52,237 - process_api - INFO - perform refresh
2024-02-09 09:14:52,438 - process_api - INFO - run step: "click_create_button"
2024-02-09 09:14:52,438 - process_api - INFO - perform click on #create
2024-02-09 09:14:54,277 - process_api - INFO - run step: "click_lookup"
2024-02-09 09:14:54,277 - process_api - INFO - perform click on dynamic-create [id='model.assetCode'] #btnLookup
2024-02-09 09:15:24,341 - process_api - ERROR - Message: 

2024-02-09 09:15:24,344 - process_api - ERROR - Traceback (most recent call last):
  File "C:\PragmaDev\ui_testing_service\process_api\__init__.py", line 82, in run
    return await self.process_runner.run_step(self, step, ctx, process, item, callback)
  File "C:\PragmaDev\ui_testing_service\process_api\process_runner.py", line 32, in run_step
    result = await function(api, step, ctx, process, item)
  File "C:\PragmaDev\ui_testing_service\process_api\modules\selenium\__init__.py", line 79, in perform
    await perform(selenium_driver, step["args"])
  File "C:\PragmaDev\ui_testing_service\process_api\modules\selenium\automation\perform.py", line 15, in perform
    element = await get_element(context, query, timeout)
  File "C:\PragmaDev\ui_testing_service\process_api\modules\selenium\automation\get.py", line 50, in get_element
    return await get_element_on_path(driver, query, timeout)
  File "C:\PragmaDev\ui_testing_service\process_api\modules\selenium\automation\get.py", line 89, in get_element_on_path
    element = wait.until(light_and_shadow_dom_callback(element, shadow_root, query))
  File "C:\PragmaDev\ui_testing_service\venv\lib\site-packages\selenium\webdriver\support\wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 


2024-02-09 09:15:24,610 - process_api - INFO - Task cd143529-bd44-41b4-a210-8839dfb77c5a completed in 00:00:32:346
